app/assets/stylesheets/darkswarm/question-mark-icon.scss

```diff
@@ -0,0 +1,58 @@
+.question-mark-icon {
+  position: relative;
+  top: 1px;
+
+  // Reset button element css attributes
+  padding: 0;
+  margin: 0;
+  width: 20px;
+  height: 20px;
+  background-color: transparent;
+
+  &:hover,
+  &:focus {
+    background-color: transparent;
+  }
+
+  &::before {
+    content: "?";
+    padding-left: 2px;
+    display: inline-block;
+    color: $tiny-blue;
+    border: 1px solid $grey-250;
+    border-radius: 50%;
+    text-align: center;
+    background-color: $grey-075;
+    width: 20px;
+    height: 20px;
+    font-weight: bold;
+    font-size: 1rem;
+  }
+}
+
+.joyride-tip-guide.question-mark-tooltip {
+  width: 16rem;
+  max-width: 65%;
+  // JS needs to be tweaked to adjust for left alignment - this is dynamic can't rewrite in CSS
+  margin-left: -7.4rem;
+  margin-top: 0.1rem;
+
+  .joyride-content-wrapper {
+    background-color: $dynamic-blue;
+    padding: 5px;
+    border-radius: 2px;
```

Comment:
it'd be great to standardize this radius, don't you think? I don't know how feasible this is though but with a quick git grep I see we use 3px quite a lot and we did the first attempt in 

https://github.com/openfoodfoundation/openfoodnetwork/blob/5fd1bc3604f50884c749bf3d2a202697cad343aa/app/assets/stylesheets/admin/globals/variables.scss#L147

although for /admin only

Reply:
I'm wondering the possibility, the avantages, the drawbacks to put a utility-first css framework (such as tailwind...) on top of the front side of the website to answer this kind of issue (using class, see : https://tailwindcss.com/docs/border-radius)

Reply:
I know we're not running last angular ... but: https://twitter.com/angular/status/1359736376581840896

Reply:
I dream about it. Tailwind made ❤️ CSS again but it feels daunting to migrate to it at this point

Reply:
I used `$radius-small` variable. 
https://github.com/openfoodfoundation/openfoodnetwork/pull/6851/files#diff-704a4699e3348b7fe0b6794127d7ac67638ca1b5394f5938a9b5cc2f2eff0035R44